### DISCUSSION TITLE
Работа над списком

### DISCUSSION THREAD

**Discussion by Re-Requeste** [totumonline/totum-ru-issues-and-discussions#466] [Как это сделать?] [answered]
Доброго времени суток. Вопрос, если для сбора данных в одну единую ячейку я использовал следующее:

`=: rowCreate(field: "header" = $header; field: "row" = $row)
	header: selectRow(table: $#ntn; fields: '*HEADER*';)
	row: selectRowList(table: $#ntn; fields: '*ALL*' )`

Как я могу обратиться к данным внутри? Т.е. если собирать по отдельности, selectRow / selecRowList, то могу обращаться к этим данным через `#test[[key]]` , а когда собираешь через rowCreate, то выдает ошибку.

**Comment by erpic-ru** [comment]
Как-то так: #test[header][key] и #test[row][[key]]  для списка

**Reply by Re-Requeste** [reply]
Спасибо, при обычном обращение заработало.
Остался еще один вопрос, когда у меня были по отдельности (selectRow/selectRowList) я мог обращаться кодом 
` listFilter(list: $train;key: "test_name" = $#value[id];)`
Но, сейчас у меня test_name лежит в header'e , как я могу обратиться к нему?

**Reply by erpic-ru** [reply, marked as answer]
Попробуйте так:
` listFilter(list: $train;key: "test_name" = $#value[id]; skip: true)`

**Reply by Re-Requeste** [reply]
Как раз читал Вашу тему на форуме, спасибо большое #160

**Reply by Re-Requeste** [reply]
Что то не понимаю где я делаю ошибку. В демо есть **"Отчет по позициям"** в insert такой код : 
`= : listReplace(list: $list; action: "summ" = $summ; action: "count" = $count; value: "value")
		list: selectRowList(table: 'tovary'; field: 'id')
	summ: listSum(list: $filter[[summa]])
	count: listCount(list: $filter[[kolichestvo]])
		filter: listFilter(list: $train; key: "pozicii_v_upd" = $#value[id])
			train: listTrain(list: $selectlist)
				selectlist: selectList(table: 'otgruzki'; field: 'dannye'; where: 'pozicii_v_upd' = $#value[id]; where: 'status_upd' = json`[4,2]`)`

Но в **"Отгрузки"**  собирает данные через selectList. 
Я же попытался объединить через rowCreate (header/all ). Но почему то не показывает данные с summ, count хотя ставил `list: $filter[row][[summa]]) и listCount(list: $filter[row][[kolichestvo]])` но значения почему то по 0. Хотя напрямую обращаюсь , все хорошо. Ошибка была еще в фильтре,  как Вы и сказали `skip:true` помогло.
